### ISSUE TITLE
Add option to retrieve new String keys from Code

### ISSUE THREAD

**Issue by Dschee** [Flinesoft/BartyCrouch#10]
BartyCrouch currently recognizes changes in Storyboards and XIBs and updates their localization strings appropriately. Adding basic support for Code files (written in Swift **or** ObjC) using the `NSLocalizedString` macro shouldn't be too hard so an addition should be considered.

The details could be more complicated of course, so maybe an alternative to NSLocalizedString may be introduced simplifying an automated update of the strings files.

**Comment by Dschee** [comment]
Work on this was started in the `work/genstrings` branch. As currently the updating logic uses `ibtool` to generate the Strings files from Interface Builder files, this branch uses the `genstrings` tool to generate Strings files from code (`.swift`, `.h` and `.m` files).

As a first step my goal is to only update the `Localizable.strings` file ignoring possibly created other String tables. Those might be added when needed by users of BartyCrouch later on in a way that is sufficient to common usage.

I also plan to make sure you can use the likes of [Laurine](https://github.com/JiriTrecak/Laurine), [SwiftGen](https://github.com/AliSoftware/SwiftGen) or [R.swift](https://github.com/mac-cain13/R.swift) with Laurine even being supported by default.

For example (using Laurines generated code style) the following line of code

```swift
public static var Large: String = "MODEL.BOARD_SIZE.LARGE".localized
```

could be configured to not be deleted like using a regex like so:

```shell
--keep "/path/to/Localizations.swift" --regex "\"([^"]*)\".localized"
```

**Comment by Dschee** [comment]
This was implemented in version 3.0.0.
